.github/scripts/test-info.mjs

```diff
@@ -530,6 +555,11 @@ function GenerateMatrix(testKind, enableCloudTests) {
             ) {
                 continue
             }
+        } else {
+            // For conformance tests, avoid running Docker and Cloud Tests together.
+            if (comp.conformance && comp.requireDocker) {
```

Comment:
I think it's ok to run Docker and Cloud  tests together. The two things aren't mutually exclusive.

Reply:
After speaking with @robertojrojas i see the benefit of having 2 separate tests